src/webrtc/call.ts

```diff
@@ -371,8 +351,8 @@ export class MatrixCall extends TypedEventEmitter<CallEvent, CallEventHandlerMap
     private candidatesEnded = false;
     private feeds: Array<CallFeed> = [];
 
-    // our transceivers for each purpose and type of media, according to SenderIndex
-    private transceivers: RTCRtpTransceiver[] = [null, null, null, null];
+    // our transceivers for each purpose and type of media
+    private transceivers = new Map<string, RTCRtpTransceiver>();
```

Comment:
I think I would still prefer either `string` being replaced by a `TransceiverType` enum or an object like

```ts
{
	[SDPStreamMetadata.Usermedia]: {
		audio: RTCRtpTransceiver,
		video: RTCRtpTransceiver,
	},
	[SDPStreamMetatada.Screenshare]: {
		audio: RTCRtpTransceiver,
		video: RTCRtpTransceiver,
	},
}
```

Reply:
Have made it a type alias, although just for a string since the 'kind' on track is intended to be other thing than just audio & video if necessary.